zerver/management/commands/send_password_reset_email.py

```diff
@@ -21,7 +21,8 @@ def add_arguments(self, parser):
         parser.add_argument('--entire-server', action="store_true", default=False,
                             help="Send to every user on the server. ")
         self.add_user_list_args(parser,
-                                help="Email addresses of user(s) to send password reset emails to.")
+                                help="Email addresses of user(s) to send password reset emails to.",
+                                all_users_arg=False)
```

Comment:
I actually intentionally removed this; with the refactor I did, that argument now works as normal with how the code is now structured.